kolibri/deployment/default/settings/base.py

```diff
@@ -132,6 +138,31 @@
 
 USE_TZ = True
 
+Q_CLUSTER = {
+    # name of the queue for this project. Since each kolibri installation's queue is self
+    # contained (since we use an isolated sqlite file), this doesn't really matter.
+    "name": "kolibriqueue",
+
+    # 3 concurrent worker processes, so 3 concurrent jobs at once
+    "workers": 3,
+
+    # 50 jobs before a worker gets reset, releasing its memory
+    "recycle": 50,
+
+    # seconds before a task is terminated. None means wait indefinitely for a task.
+    "timeout": None,
+
+    # number of successful tasks we save to the DB. 0 means save all. All failed tasks are saved.
+    "save_limit": 0,
+
+    # catch_up, when True, makes the workers catch up to all scheduled tasks that elapsed while it was down.
+    "catch_up": False,
```

Comment:
Would be interesting to revisit this, or look at other approaches, for things like syncing.